### PR TITLE
WallpaperContainer: style selection check with CSS

### PR DESCRIPTION
**Pull request by danrabbit**
Use CSS instead of an image for styling the selection check. This makes sure we get proper background states, darker borders in dark mode etc.

### CODE REVIEW

File: src/Widgets/WallpaperContainer.vala [hunk 1]
```diff
@@ -91,12 +91,15 @@ public class PantheonShell.WallpaperContainer : Gtk.FlowBoxChild {
         card_box.add (image);
         card_box.margin = 9;
 
-        var check = new Gtk.Image.from_resource ("/io/elementary/switchboard/plug/pantheon-shell/check.svg") {
+        var check_provider = new Gtk.CssProvider ();
+        check_provider.load_from_resource ("/io/elementary/switchboard/plug/pantheon-shell/Check.css");
+
+        var check = new Gtk.RadioButton (null) {
```
Comment (jeremypw) [inline src/Widgets/WallpaperContainer.vala:97]: Gtk.RadioButton is deprecated in Gtk4 I think.  Is there a solution that will work in Gtk4 as well?

**Comment by jeremypw** [comment]
@danrabbit We did it with an image to avoid using the deprecated Gtk.RadioButton.  Is there another solution that also works in Gtk4?

**Comment by danrabbit** [comment]
@jeremypw I wouldn't avoid using radios in Gtk 3 since it's pretty trivial to change to a check button in Gtk 4.

In general using images isn't going to give good results because there are a lot of widget states to cover and it isn't going to look very good with text scaling

**Comment by jeremypw** [comment]
OK, I'll merge this then.

**Review by jeremypw — APPROVED** [review]

**Comment by lenemter** [comment]
When window is out of focus, the radio button is black, is that intentional?

**Comment by jeremypw** [comment]
Shouldn't think so - I didn't notice that :disappointed: @danrabbit can this be fixed?

**Comment by jeremypw** [comment]
@lenemter For me the radio button is greyed out which is what is expected (in the Gtk3 version at least)

**Comment by danrabbit** [comment]
Yeah it should be grey in backdrop state. If you're seeing black could you open a new issue report with more details?

**Comment by lenemter** [comment]
Oh, in dark style it is very dark gray, so it looks like black on a first glance

**Comment by danrabbit** [comment]
@lenemter ah yeah I see that. Can you open an issue with the stylesheet? I think I would consider that a bug in the stylesheet since it effects all checks and radios

**Pull request merged**